src/renderer/components/utils/menu/AccessibleMenu.tsx

```diff
@@ -82,6 +80,8 @@ export default class AccessibleMenu extends React.Component<Props, State> {
             && this.state.triggerElem
         ) {
             if (this.state.triggerElem != null) {
+                // WHY ?
+                // @ts-ignore
```

Comment:
Simply change the line `triggerElem: document.activeElement` to `triggerElem: document.activeElement as HTMLElement`, and `triggerElem: Element | undefined;` to `triggerElem: HTMLElement | undefined;`, then remove the `@ts-ignore` comment.

Reply:
but why force cast Element to HTMLElement, is the typing broken ?

Reply:
`document.activeElement` is a DOM property (HTML, SVG, XML, etc.)
https://developer.mozilla.org/en-US/docs/Web/API/DocumentOrShadowRoot/activeElement
https://developer.mozilla.org/en-US/docs/Web/API/Document

But in our case, we know that the Thorium UI is built with HTML markup (+ a few SVG images), so we can cast the correct type for better compile-time checking.

Reply:
bfea739

thank you Daniel 👍